Corona-Warn-App/src/main/res/values/covid_certificate_strings.xml

```diff
@@ -393,4 +399,41 @@
     <!-- XHED: Cov Pass Info Fragment title -->
     <string name="cov_pass_info_section_three">"Please note that third parties in other countries may use different apps to verify certificates."</string>
 
+    <!-- #####################################################
+        CONFIRMED STATUS CARD
+        ##################################################### -->
+
+    <!-- XTXT: Confirmed status card title -->
+    <string name="confirmed_status_title">"Status-Nachweis"</string>
+
+    <!-- XTXT: Confirmed status card 2g+ PCR subtitle -->
+    <string name="confirmed_status_2g_pcr_subtitle">"2G+ PCR-Test"</string>
+    <!-- XTXT: Confirmed status card 2g+ RAT subtitle -->
+    <string name="confirmed_status_2g_rat_subtitle">"2G+ Schnelltest"</string>
+
+    <!-- XTXT: Confirmed status card 2G badge -->
+    <string name="confirmed_status_2g_badge">"2G"</string>
+    <!-- XTXT: Confirmed status card 2G+ badge -->
+    <string name="confirmed_status_2g_plus_badge">"2G+"</string>
+    <!-- XTXT: Confirmed status card 3G badge -->
+    <string name="confirmed_status_3g_badge">"3G"</string>
+    <!-- XTXT: Confirmed status card 3G+ badge -->
+    <string name="confirmed_status_3g_plus_badge">"3G+"</string>
+
+    <!-- XTXT: Confirmed status card 2G body -->
+    <string name="confirmed_status_2g_body">"Ihre Zertifikate ermöglichen Ihnen Zutritt für 2G Events. Wenn Sie Ihren aktuellen Status vorweisen müssen, schließen sie diese Ansicht und zeigen Sie den QR-Code auf der Zertifikatsübersicht."</string>
```

Comment:
Change:
 Ihre Zertifikate ermöglichen Ihnen Zutritt für 2 G Events 
 --> 
Ihre Zertifikate erfüllen die 2G Regel.

Reply:
Thanks, I see you have already pushed the new strings yourself. Can you please approve?